Makefile

```diff
@@ -204,3 +204,29 @@ migrate: migrate-lms migrate-cms
 # Part of https://github.com/openedx/wg-developer-experience/issues/136
 ubuntu-requirements: ## Install ubuntu 22.04 system packages needed for `pip install` to work on ubuntu.
 	sudo apt install libmysqlclient-dev libxmlsec1-dev
+
+test-eslint:
+	python scripts/quality_test/quality_test.py eslint
+
+test-stylelint:
+	python scripts/quality_test/quality_test.py stylelint
+
+test-xsslint:
+	python scripts/quality_test/quality_test.py xsslint
+
+test-lint:
+	pycodestyle .
+
+test-pi_check:
+	python scripts/quality_test/quality_test.py pii_check
+
+test-check_keyword:
+	python scripts/quality_test/quality_test.py check_keywords
+
+test-js:
+	 xvfb-run --auto-servernum python scripts/quality_test/js_test.py --option jstest
```

Comment:
is xvfb-run necessary, or could this just be `python scripts/quality_test/js_test.py --option jstest`?